plugins/nmap/help.md

```diff
@@ -15,7 +15,7 @@ _This plugin does not contain any requirements._
 
 # Supported Product Versions
 
-* 2022-11-18
+_There are no supported product versions listed._
```

Comment:
did you remove this intentionally?

Reply:
No thats my bad it did that when I regenerated